tests/integration/models/hostapp.spec.ts

```diff
@@ -0,0 +1,75 @@
+// tslint:disable-next-line:import-blacklist
+import * as _ from 'lodash';
+import * as m from 'mochainon';
+import { balena } from '../setup';
+import type * as BalenaSdk from '../../..';
+const { expect } = m.chai;
+
+const containsVersion = (
+	versions: BalenaSdk.OsVersion[],
+	expected: Partial<BalenaSdk.OsVersion>,
+) => {
+	const os = _.find(versions, expected);
+	expect(os).to.not.be.undefined;
+};
+
+describe.only('Hostapp model', function () {
```

Comment:
Forgotten `.only` :)

Reply:
🤦  we need to add a linting rule for this :)

Reply:
Added in
See: https://github.com/balena-io/balena-sdk/pull/876